docs/getting-started.md

```diff
@@ -290,6 +295,6 @@ update the build files with your changes.
 #### Build.bazel build failures when switching branches
 
 In case you work on two or more branches, `make generate` for example might fails,
```

Comment:
Since you're already proposing changes to the doc, could you also fix the `... might fails`? it should be `... might fail`

Reply:
Done